shell/platform/android/image_external_texture_vk.cc

```diff
@@ -88,6 +141,7 @@ void ImageExternalTextureVK::ProcessFrame(PaintContext& context,
   }
 
   dl_image_ = impeller::DlImageImpeller::Make(texture);
+  AddImage(dl_image_, key);
   CloseHardwareBuffer(hardware_buffer);
```

Comment:
I'm not sure how/if we should still be calling close here.

Reply:
Moving another CloseHWB/CloseImage above silenced some of these errors.

Reply:
After we've made the DlImage we should close the hardware buffer. The cache shouldn't change that as we already close the hardware buffer immediately after we create the DlImage in the old code.